soyuz_project/settings/local.py

```diff
@@ -10,6 +10,9 @@
     "DJANGO_SECRET_KEY",
     default="cg#p$g+j9tax!#a3cup@1$8obt2_+&k3q+pmu)5%asj6yjpkag",
 )
+
+HUBSPOT_API_KEY = env("HUBSPOT_API_KEY")
```

Comment:
instead of this being in local and production I think it can go in base?

https://github.com/rocketacademy/soyuz/blob/main/soyuz_project/settings/base.py